src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -72,20 +88,62 @@ def test_aks_create_default_service(self, resource_group, resource_group_locatio
             self.check('agentPoolProfiles[0].count', 5)
         ])
 
+        # scale down
+        self.cmd('aks scale -g {} -n {} -c 1'.format(resource_group, aks_name), checks=[
+            self.check('agentPoolProfiles[0].count', 1)
+        ])
+
+        # show again
+        self.cmd('aks show -g {} -n {}'.format(resource_group, aks_name), checks=[
+            self.check('agentPoolProfiles[0].count', 1)
+        ])
+
+        # delete
+        self.cmd('aks delete -g {} -n {} -y'.format(resource_group, aks_name), checks=[
+            self.is_empty()
+        ])
+
+        # show again and expect failure
+        self.cmd('aks show -g {} -n {}'.format(resource_group, aks_name), expect_failure=True)
+
+    @ResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus')
+    @RoleBasedServicePrincipalPreparer()
+    def test_aks_create_service_no_wait(self, resource_group, resource_group_location, sp_name, sp_password):
+        ssh_pubkey_file = self.generate_ssh_keys().replace('\\', '\\\\')
+        aks_name = self.create_random_name('cliakstest', 16)
+        dns_prefix = self.create_random_name('cliaksdns', 16)
+
+        # create --no-wait
+        create_cmd = 'aks create -g {} -n {} -p {} --ssh-key-value {} -l {} ' \
```

Comment:
I would recommend the more readable kwargs syntax over the `.format` syntax.  It would look something like:
```Python
self.kwargs.update({
  'a': blah,
  'b': blah,
  'c': blah
})
self.cmd('mycommand --foo {a} --blah {b} --boo {c}',
              checks=self.check('fooProperty', '{a}'))
```